api/explored/requests.go

```diff
@@ -0,0 +1,39 @@
+package explored
```

Comment:
my preferred file layout for api stuff is:

- request/response types in `api.go`
- server implementation in `server.go`
- client implementation in `client.go`

Reply:
yeah, should change all of them